draft-ietf-rats-corim.md

```diff
@@ -303,6 +303,16 @@ convention - e.g., {{Section 4.4.1.5 of -psa-token}}).
 {::include cddl/digest.cddl}
 ~~~
 
+### Tagged Bytes Type {#sec-common-tagged-bytes}
+
+An opaque, variable-length byte string.
+It can be used in different contexts: as an instance, class or group identifier in an `environment-map`; as a raw value measurement in a `measurement-values-map`.
```

Comment:
```suggestion
It can be used in different contexts: as an instance, a class or a group identifier in an `environment-map`; as a raw value measurement in a `measurement-values-map`.
```

Reply:
Agreed to omit this suggestion.